core/src/main/java/com/graphhopper/routing/ProfileResolver.java

```diff
@@ -42,14 +39,13 @@
  */
 public class ProfileResolver {
     private final EncodingManager encodingManager;
-    private final List<ProfileConfig> profiles;
+    private final Map<String, ProfileConfig> profilesByName;
```

Comment:
We are not using this map (anymore?), can we just revert this change? I would prefer to keep the profiles' order.

Reply:
Sure, I accidentally only partially reverted the change.